src/utils/spo.ts

```diff
@@ -321,6 +321,31 @@ export const spo = {
     });
   },
 
+  getSpoGraphSiteId(webUrl: string, logger: Logger, debug: boolean): Promise<string> {
+    return new Promise<string>((resolve: (spoUrl: string) => void, reject: (error: any) => void): void => {
```

Comment:
Hi @MathijsVerbeeck, saw this on first sight. Could you refactor the util to use `async/await` instead of promises please?

Reply:
Woops. My bad!